app/dcwrapper/api.py

```diff
@@ -0,0 +1,260 @@
+import requests
+from app.job import rq
+import time
+import json
+import os 
+import urllib
+import zipfile
+from app.models import db, DataSet, Bash
+from app.job import download
+from app.bash import bash_helper
+from rq import Connection, Worker
+
+
+HEADERS = {'Content-Type': 'application/json', 'X-Api-Key': 'mint-data-catalog:e038e64c-c950-4fbc-9070-a3e7138b6c4f:dce8a09a-200e-43ca-b996-810c2c437d3a', 'cache-control': 'no-cache', 'Postman-Token': '3084e843-b082-4bfb-be1a-bb4ac72c865'}
```

Comment:
It's a session. not permanent.
request https://api.mint-data-catalog.org/get_session_token first.

Check https://docs.google.com/document/d/1I3CjYB-GDdFTZO-dHsHB0B5f0iEbzHnr8QHSKN5k3Sc/edit#heading=h.6uvldlcr99e1

Reply:
'X-Api-Key'